### PR TITLE
Created a blog about IstioCon 2021

### PR DESCRIPTION
**Pull request by macruzbar**
**This PR is a blogpost about IstioCon 2021, for the istio.io blog.**

[ ] Configuration Infrastructure
[X ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by macruzbar** [comment]
can't seem to properly add the image, but here it is: https://github.com/macruzbar/istio.io/blob/master/content/en/blog/2020/istioconlogo.jpg

**Comment by linsun** [comment]
/ok-to-test

LGTM thanks @macruzbar!  For the image,  you'd need to check it in to your branch.

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Join us for the first IstioCon in 2021!
```
Comment (frankbu) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:1]: You need to format using standard headers like all Istio blogs.
```suggestion
---
title: "Join us for the first IstioCon in 2021!"
description: The inaugural conference for Istio will take place at the end of February.
publishdate: 2020-12-03
attribution: "Istio Steering Committee"
keywords: [istioCon,Istio,conference]
---
```

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+IstioCon 2021 will be the inaugural conference for Istio, the industry's most popular service mesh. In its inaugural year, IstioCon will be 100% virtual, connecting community members across the globe with Istio's ecosystem. This conference will take place at the end of February. 
+
+![/macruzbar/istio.io/content/en/blog/2020/istioconlogo.jpg]
```
Comment (frankbu) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:5]: Put a copy of the image in `/blog/2020/istiocon-2021/istioconlogo.jpg` and then reference it here:
```suggestion
{{< image width="75%"
    link="./istioconlogo.jpg"
    alt="IstioCon logo"
    >}}
```
You'll need to look at the preview to decide what `image width="75%"` is appropriate.

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md (hunk 1 above)
Comment (frankbu) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md:1]: Also, you need to rename this file from `istiocon-2021/istiocon-2021.md` -> `istiocon-2021/index.md`.

**Comment by frankbu** [comment]
Technically, this should be a news entry, not a blog (see https://istio.io/latest/about/contribute/add-content/#content-types), but we don't currently have a slot in the news section for this kind of news.

cc @craigbox

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+All the information related to IstioCon will be published on the [coference website](https://events.istio.io/). At this time, we encourage Istio users, developers, partners and advocates to [submit a session proposal through the conference's CFP](https://sessionize.com/istiocon-2021/). IstioCon will be an opportunity to showcase the lessons learned from running Istio in production, hands-on experiences from the Istio community, and featuring maintainers from across the Istio ecosystem. The conference offers a mix of keynotes, technical talks, lightning talks, workshops and roadmap sessions. Choose from the following formats to submit a session for IstioCon:
```
Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: change to conference (add 'n')

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: partners, and (we'll use serial commas)

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: I'd change to "through the conference's CFP portal."

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: change to "conference website" (add 'n')

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: change to "partners, and" (serial comma)

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: I'd change to "through the conference's CFP portal"

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: change to "IstioCon provides an opportunity"

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: change to "and will feature"

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: add serial comma "workshops, and"

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: change to "session proposal for"

Comment (lyriccoloratura) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: I would move the sentence "At this time, we encourage Istio users, developers, partners and advocates to [submit a session proposal through the conference's CFP](https://sessionize.com/istiocon-2021/)." down to in between the end of the "across the Istio ecosystem." sentence and the beginning of the "The conference offers a mix of keynotes" sentence.

**Comment by craigbox** [comment]
I find the distinction between "news" and "blog" to be arbitrary, especially given "news" Is only used to refer to release-related things.  Do we know what the reason was for having them separate?  Does it hold true going forward?

There's separate RSS feeds for "news", "blog" and "news and blog" - I'm not sure if we can get stats on how many people subscribe to one or the other?  I know it's confused me more than once.  It would be interesting to see any metrics on this (perhaps hits to those pages?)

For now I think we should make this topic a blog post, and Docs WG can evaluate if there's value in the above as time permits.

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md (hunk 3 above)
Comment (craigbox) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md:7]: FYI, you can propose changes like this yourself by hitting the little "±" button to the right of "Preview" in the comment box: https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Join us for the first IstioCon in 2021!
+
+IstioCon 2021 will be the inaugural conference for Istio, the industry's most popular service mesh. In its inaugural year, IstioCon will be 100% virtual, connecting community members across the globe with Istio's ecosystem. This conference will take place at the end of February.
```
Comment (craigbox) [inline content/en/blog/2020/istiocon-2021/istiocon-2021.md:3]: Which industry?

We could say "the world's" or similar, and refer to the CNCF 2020 survey which backs up the claim?

**Comment by frankbu** [comment]
> I find the distinction between "news" and "blog" to be arbitrary, especially given "news" Is only used to refer to release-related things. Do we know what the reason was for having them separate? Does it hold true going forward?

@craigbox We used to have just one list for everything, but we noticed that the technical articles where getting lost in all the release related noise. We also realized that the release info could be better organized, instead of just a single list of entries. So more than a year ago we split it into news and blogs, with the intent that blogs would just be technical articles, although it seems that it has still been a bit of a mix of news and articles since then.

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md (hunk 1 above)
Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: added and changed!

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md (hunk 3 above)
Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: added!

Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: added!

Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: changed

Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: added!

Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: changed

Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: added

Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: added

Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: moved

**Comment by ericvn** [comment]
@macruzbar Note that you can click on Details to the right of the failing test, and then click on the logs. I've copied part of that output here:
```
level=info msg="Execution took 1m56.490895461s"
    ./en/blog/2020/istiocon-2021/index.md
        1 | # Join us for the first IstioCon in 2021! 
        3 | IstioCon 2021 will be the inaugural co 
        3 |  mesh. In its inaugural year, IstioCon will be 100% virtual, connect 
        5 | ![/macruzbar/istio.io/content/en/blog/2020 
        5 | istio.io/content/en/blog/2020/istioconlogo.jpg] 
        7 | ll the information related to IstioCon will be published on the [cof 
        7 | Con will be published on the [coference website](https://events.istio 
        7 | ssionize.com/istiocon-2021/). IstioCon will be an opportunity to sho 
        7 | rmats to submit a session for IstioCon: 
        9 | * **Presentation:** 40-minute presentation, maximum of 2 sp 
       10 | l:** 40 minutes of discussion amongst 3 to 5 speakers 
       11 | * **Workshop:** 160-minute (2h 40m), in-depth, hands-on  
       11 | * **Workshop:** 160-minute (2h 40m), in-depth, hands-on pres 
       11 |  **Workshop:** 160-minute (2h 40m), in-depth, hands-on presenta 
       12 | * **Lighting Talk:** 10-minute presentation, limited to 1 sp 
       16 |  you can join us at the first IstioCon in 2021! 

>> 16 spelling errors found in 611 files
To learn how to address spelling errors, please see https://istio.io/about/contribute/creating-and-editing-pages/#linting
./en/blog/2020/istiocon-2021/index.md:1: MD002 First header should be a top level header
./en/blog/2020/istiocon-2021/index.md:3: MD009 Trailing spaces
./en/blog/2020/istiocon-2021/index.md:11: MD009 Trailing spaces
./en/blog/2020/istiocon-2021/index.md:14: MD009 Trailing spaces
./en/blog/2020/istiocon-2021/index.md:1: MD026 Trailing punctuation in header
```

These are the changes needed to pass the lint test. It's hard to tell the misspellings in that output (it's usually close to the middle of the line). You can run `make lint-fast` if you have docker installed to run the test on your computer and the misspellings will be highlighted.

Note also, you can click on Details to the right of the netlify task and it will bring up a website with your changes. You can go to your new page and see what it would look like.

**Comment by ericvn** [comment]
@macruzbar Note that if you also add a new commit, all the tests will automatically rerun so you don't need to start test manually. You should only need to run a test manually if there is a test flake (it occasionally fails).

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md (hunk 2 above)
Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: I changed the syntax. I can see the file added in the folder on my repo here: https://github.com/macruzbar/istio.io/tree/cf63f6fc1d809cb8668a830e67c69735844f82dd/content/en/blog/2020/istiocon-2021 but I'm not sure it is part of the batch I submitted on this PR.

File: content/en/blog/2020/istiocon-2021/istiocon-2021.md (hunk 4 above)
Comment (macruzbar) [reply content/en/blog/2020/istiocon-2021/istiocon-2021.md]: I added a link to the survey. The wording was decided in collaboration with Srinath and Jacinda.

File: content/en/blog/2020/istiocon-2021/index.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Join us for the first IstioCon in 2021!"
+description: The inaugural conference for Istio will take place at the end of February.
+publishdate: 2020-12-03
```
Comment (frankbu) [inline content/en/blog/2020/istiocon-2021/index.md:4]:
```suggestion
publishdate: 2020-12-04
```

File: content/en/blog/2020/istiocon-2021/index.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+attribution: "Istio Steering Committee"
+keywords: [istioCon,Istio,conference]
+---
+
+IstioCon 2021 will be the inaugural conference for Istio, the industry's [most popular service mesh](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf). In its inaugural year, IstioCon will be 100% virtual, connecting community members across the globe with Istio's ecosystem. This conference will take place at the end of February. 
+
+{{< image width="75%"
+    link="/istioconlogo.jpg"
```
Comment (frankbu) [inline content/en/blog/2020/istiocon-2021/index.md:12]:
```suggestion
    link="./istioconlogo.jpg"
```

File: content/en/blog/2020/istiocon-2021/index.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Join us for the first IstioCon in 2021!"
+description: The inaugural conference for Istio will take place at the end of February.
+publishdate: 2020-12-03
+attribution: "Istio Steering Committee"
+keywords: [istioCon,Istio,conference]
```
Comment (frankbu) [inline content/en/blog/2020/istiocon-2021/index.md:6]:
```suggestion
keywords: [IstioCon,Istio,conference]
```

**Comment by frankbu** [comment]
@macruzbar If you'd like to add a announcement banner on the website (maybe closer to the actual conference date), you can also create an event entry similar to this https://github.com/istio/istio.io/blob/master/content/en/events/banners/kubecon-america-2019.md

File: content/en/blog/2020/istiocon-2021/index.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+---
+title: "Join us for the first IstioCon in 2021!"
+description: The inaugural conference for Istio will take place at the end of February.
+publishdate: 2020-12-04
```
Comment (frankbu) [inline content/en/blog/2020/istiocon-2021/index.md:4]: This should be updated to the date when the PR is ready to merge. If it's ready today, then:
```suggestion
publishdate: 2020-12-08
```

**Comment by frankbu** [comment]
@pedrogk have you signed the CLA? https://github.com/istio/istio.io/pull/8609#issuecomment-740730822

**Comment by macruzbar** [comment]
@googlebot I consent

**Review by frankbu — APPROVED** [review]

**Pull request merged**